trelloscrum.js

```diff
@@ -617,7 +617,7 @@ function ListCard(el, identifier){
 		clearTimeout(to);
 
 		to = setTimeout(function(){
-			var $title=$card.find('a.list-card-title');
+			var $title=$card.find('span.list-card-title');
```

Comment:
@gjdycus I think using `.js-card-name` class is better.

```js
$title=$card.find('.js-card-name');
```

This problem is caused by using HTML selector in JavaScript codes.
JavaScript codes should better to use `js-` prefix class because Trello developers also use `js-` prefix class from their js.